server/settings/creator_plugins.py

```diff
@@ -164,7 +175,20 @@ class FolderCreationConfigModel(BaseSettingsModel):
         title="Enabled folder creation",
         default=False,
     )
-
+    folder_create_type: str = SettingsField(
```

Comment:
@jakubjezek001 @robin-ynput is changing of these attributes order ok for you?

Reply:
I'm not sure tbh.. if I understand correctly this change, we go from:
```
folder regexes
default folder type
--
task regexes
default task type
```
to 
```
default folder type
default task type
folder regexes
task regexes
```
Purely personal preferences here, but I think I'd prefer keeping them grouped per folder/task instead. 
Not a biggie if other people disagree.

Reply:
Thank you for pointing that out. 
The original order is used while enhancing it with a section argument. 
https://github.com/ynput/ayon-traypublisher/pull/44/commits/2b496aa910cd3d1fb4a9543bdfbb86bb2df25069